pkg/virtctl/guestfs/guestfs.go

```diff
@@ -0,0 +1,374 @@
+package guestfs
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/spf13/cobra"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/kubectl/pkg/cmd/attach"
+
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+
+	"kubevirt.io/kubevirt/pkg/virtctl/templates"
+)
+
+const (
+	defaultRegistry  = "registry:5000/kubevirt"
+	defaultImageName = "libguestfs-tools"
+	defaultTag       = "latest"
```

Comment:
I'm not sure how to set these values, and I'd like some help here. I don't think they can be hardcoded